src/main/java/sg/ncl/testbed_interface/TeamPageApplyTeamForm.java

```diff
@@ -23,6 +22,8 @@
     
     @NotNull(message="Please select one of them")
     private String isPublic = TeamVisibility.PUBLIC.toString();
+
+    private boolean isClass = false;
     
     public TeamPageApplyTeamForm() {   
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)